packages/modules/Extension-Ads/src/contentScript/content.js

```diff
@@ -858,12 +890,27 @@ if ( typeof vAPI === 'object' && !vAPI.contentScript ) {
       debugLog.content && console.log(`extractContent: ${frameId}`,elt?elt:"",content,document)
       return content;
     },
-
-    getAdsCoordinates(element) {
+    async getAdsCoordinates(element) {
       // for iframe elements
       if (typeof element.getBoundingClientRect !== 'function') {
-        return null;
-      }
+        const src = window.location.href;
```

Comment:
You don't need to send it, because you can get iframe url from event.source: source.location.href